src/applications/mhv/medical-records/containers/Allergies.jsx

```diff
@@ -33,7 +37,76 @@ const Allergies = () => {
     [dispatch],
   );
 
-  const download = () => {};
+  const generateAllergiesPdf = async res => {
+    const pdfData = {
+      headerLeft: 'Roberts, Jesse',
```

Comment:
ooc can the name/dob be filled dynamically? Or is that covered in a follow-on?

Reply:
I'll have to check with Mike about whether or not we can fill those fields with real data yet. @mmoyer-va

Reply:
I added name and dob from the redux store

Reply:
nice!